string functionName
---
int64 returnVal